internal/handlers/common/handler.go

```diff
@@ -0,0 +1,111 @@
+	MsgDropDatabase(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgGetCmdLineOpts returns a summary of all runtime and configuration options.
+	MsgGetCmdLineOpts(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgGetLog returns the most recent logged events from memory.
+	MsgGetLog(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgGetParameter returns the value of the parameter.
+	MsgGetParameter(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgHostInfo returns a summary of the system information.
+	MsgHostInfo(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgIsMaster returns the role of the FerretDB instance.
+	MsgIsMaster(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgHello returns the role of the FerretDB instance.
+	MsgHello(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgListCollections returns the information of the collections and views in the database.
+	MsgListCollections(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgListDatabases returns a summary of all the databases.
+	MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgPing returns a pong response. Used for testing purposes.
+	MsgPing(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgServerStatus returns an overview of the databases state.
+	MsgServerStatus(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgWhatsMyURI an internal command.
+	MsgWhatsMyURI(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgCount returns the count of documents that's matched by the query.
+	MsgCount(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgCreateIndexes creates indexes on a collection.
+	MsgCreateIndexes(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgDelete deletes documents matched by the query.
+	MsgDelete(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgFind returns documents matched by the query.
+	MsgFind(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgFindAndModify inserts, updates, or deletes, and returns a document matched by the query.
+	MsgFindAndModify(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgInsert inserts documents into the database.
+	MsgInsert(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgUpdate updates documents that are matched by the query.
+	MsgUpdate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgDebugError used for debugging purposes.
+	MsgDebugError(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgDebugPanic used for debugging purposes.
+	MsgDebugPanic(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg, error)
+
+	// MsgQueryCmd runs query operation command.
+	MsgQueryCmd(ctx context.Context, query *wire.OpQuery) (*wire.OpReply, error)
```

Comment:
```suggestion
	// QueryCmd runs query operation command.
	QueryCmd(ctx context.Context, query *wire.OpQuery) (*wire.OpReply, error)
```

`Msg` prefix comes from OP_MSG, but that's OP_QUERY